lib/active_shipping/rate_estimate.rb

```diff
@@ -66,13 +66,19 @@ module ActiveShipping
   # @!attribute delivery_category
   #   The general classification of the delivery method
   #   @return [String]
+  #
+  # @!attribute shipment_options
+  #   Additional priced options bundled with the given rate estimate with price in cents
+  #   @return [Array<{ code: String, price: Integer }>]
```

Comment:
Certain this is all you'll need? No convenience methods or checks that would merit an `ActiveShipping::ShippingOption` class? Using an array of hashes in this case is in line with what exists in this gem. But just making sure we are forward thinking.